tests/pages/base.py

```diff
@@ -5,21 +5,21 @@
 from selenium.webdriver.common.by import By
 from selenium.webdriver.support.select import Select
 
-from page import Page, PageRegion
+from pypom import Page, Region
 from regions.newsletter import NewsletterEmbedForm
 
 
 class BasePage(Page):
 
-    _url = '{base_url}/{locale}'
+    URL_TEMPLATE = '/{locale}'
 
-    def __init__(self, base_url, selenium, locale='en-US', **kwargs):
-        super(BasePage, self).__init__(base_url, selenium, locale=locale, **kwargs)
+    def __init__(self, selenium, base_url, locale='en-US', **url_kwargs):
+        super(BasePage, self).__init__(selenium, base_url, locale=locale, **url_kwargs)
 
     def wait_for_page_to_load(self):
         super(BasePage, self).wait_for_page_to_load()
-        el = self.find_element((By.TAG_NAME, 'html'))
-        self.wait.until(lambda s: 'loaded' in el.get_attribute('class'))
+        self.wait.until(lambda s: self.is_element_present(
```

Comment:
Ok understood - I guess we'll see if this change has any side effects, but hopefully it will be more robust than before.